homeassistant/components/glances/sensor.py

```diff
@@ -321,3 +338,26 @@ def native_value(self) -> StateType:
                 StateType, value[self._sensor_label][self.entity_description.key]
             )
         return cast(StateType, value[self.entity_description.key])
+
+
+class GlancesTimestampSensor(GlancesSensor):
+    """Implementation of a Glances Timestamp sensor."""
+
+    _attr_native_value: datetime | None = None
+
+    @property
+    def native_value(self) -> StateType | datetime:
```

Comment:
Same comment as on the previous PR https://github.com/home-assistant/core/pull/103168#discussion_r1404558026

Reply:
Sure, I'll have a look at this

Reply:
I moved the uptime calculation in the DataUpdateCoordinator as requested, let me know if this looks better.
Thanks !